### PR TITLE
Junesh k patch 1

### PR DESCRIPTION
**Pull request by JuneshK**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4742"></a>

### CODE REVIEW

**Review by krshkun — CHANGES_REQUESTED** [review]
Please delete the extra file

File: data/JuneshK.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/channel/UCburH8LA-mVat8dW9VVoUxw",
+      "icon": "FaYoutube"
+    }
+
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/junesh-k-451a0b205",
+      "icon": "FaLinkedIn"
```
Comment (krshkun) [inline data/JuneshK.json:22]:
```suggestion
      "icon": "FaLinkedin"
```

Comment (JuneshK) [reply data/JuneshK.json:22]: Noted.

File: data/JuneshK.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{
+  "name": "Junesh Kumar",
+  "type": "personal",
+  "bio": "Software Engineer . Currently in my Sophomore Year . Youtuber ",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/JuneshK1",
+      "icon": "FaTwitter"
+    }
```
Comment (SaraJaoude) [inline data/JuneshK.json:11]:
```suggestion
    },
```

File: data/JuneshK.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/channel/UCburH8LA-mVat8dW9VVoUxw",
+      "icon": "FaYoutube"
+    }
```
Comment (SaraJaoude) [inline data/JuneshK.json:17]:
```suggestion
    },
```

File: data/JuneshK.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/junesh-k-451a0b205",
+      "icon": "FaLinkedin"
+    }
```
Comment (SaraJaoude) [inline data/JuneshK.json:23]:
```suggestion
    },
```

File: data/JuneshK.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+    {
+      "name": "GitHub",
+      "url": "https://github.com/JuneshK",
+      "icon": "FaGithub"
+    }
```
Comment (SaraJaoude) [inline data/JuneshK.json:29]:
```suggestion
    },
```

**Pull request merged**